src/components/implementation/tests/unit_pingshmem/Makefile

```diff
@@ -0,0 +1,18 @@
+# Required variables used to drive the compilation process. It is OK
+# for many of these to be empty.
+#
+# The set of interfaces that this component exports for use by other
+# components. This is a list of the interface names.
+INTERFACE_EXPORTS =
+# The interfaces this component is dependent on for compilation (this
+# is a list of directory names in interface/)
+INTERFACE_DEPENDENCIES = init capmgr pongshmem
```

Comment:
I might be missing something, but shouldn't this be `memmgr` rather than `capmgr`? I think you're only depending on `memmgr` interfaces.